test/forge/libs/DelegationPool.t.sol

```diff
@@ -57,17 +57,18 @@ contract DelegationPoolTest is Test {
     function testDistributeReward_AmountZero() public {
         vm.record();
 
-        delegationPoolLibUser.distributeReward(address(delegationPoolLibUser), 0);
+        delegationPoolLibUser.distributeReward(0);
 
         // did not write to storage
         (, bytes32[] memory writes) = (vm.accesses(address(this)));
         assertEq(writes.length, 0);
     }
 
-    function testCannotDistributeReward_NoTokensDelegated() public {
-        vm.expectRevert(abi.encodeWithSelector(NoTokensDelegated.selector, (address(delegationPoolLibUser))));
-        delegationPoolLibUser.distributeReward(address(delegationPoolLibUser), 1);
-    }
+    // In our version of the contracts we cannot hit this case
```

Comment:
Add a test that tries to distribute rewards where the delegator pool is empty, so if something is changed in the wrong way we can see the test is failing